.eslintrc.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
     },
   ],
   rules: {
+    '@typescript-eslint/prefer-nullish-coalescing': 'off',
```

Comment:
It looks like this is being set in the wrong place. This `rules` property is for the _base_ rules, applied to the entire repository. The TypeScript rule you're trying to disable here is set in an override block, so that it only applies to TypeScript files. The overrides will override anything set here.

You can disable this by moving this into a `rules` block _inside_ the TypeScript override block

Reply:
Though... I am curious, why disable this rule?

Reply:
Cool, makes sense